config/environments/migration.rb

```diff
@@ -4,6 +4,13 @@
   config.log_level = :debug
   config.ssl_options = { redirect: { exclude: ->(request) { request.path.include?("/healthcheck") } } }
 
+  config.action_controller.perform_caching = true
```

Comment:
Does the migration environment not have Redis already setup for caching? I think we should keep it as close to prod as possible

Reply:
we don't have Redis in NPQ reg, adding a ticket to add it